authorizations/authorization-ranger/build.gradle.kts

```diff
@@ -32,13 +32,10 @@ dependencies {
     exclude(group = "*")
   }
   implementation(libs.bundles.log4j)
-  implementation(libs.commons.collections4)
   implementation(libs.commons.lang3)
   implementation(libs.guava)
```

Comment:
I think the jars like log4j, guava are already provided by Gravitino's base classpath, so we can remove these when doing copyLib, you can refer to @yuqi1129 's PR #4521